2021-12-17 17:36:37,129 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Enable SLS, effective endpoint: http://cn-shanghai.log.aliyuncs.com
2021-12-17 17:36:37,359 [529] [WARNING ] [benchmark_blade_resnet50_v1.py:182] Eas daemon is offline
2021-12-17 17:36:37,420 [529] [WARNING ] [benchmark_blade_resnet50_v1.py:182] Failed to import torch
2021-12-17 17:36:37,420 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] is_saved_model_pb: False
2021-12-17 17:36:37,420 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Start to load frozen pb from /root/inference-optimization-benchmark/models/frozen_resnet50_v1.pb
2021-12-17 17:36:38,218 [529] [WARNING ] [benchmark_blade_resnet50_v1.py:182] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops
2021-12-17 17:36:38,867 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Found frozen_resnet50_v1.pb. Proceed with optimization.
2021-12-17 17:36:38,867 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Begin of combine_info_with_blade_config
2021-12-17 17:36:38,867 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Combine frozen pb info with blade config
2021-12-17 17:36:38,872 [529] [WARNING ] [deprecation.py:339] From benchmark_blade_resnet50_v1.py:182: remove_training_nodes (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.graph_util.remove_training_nodes`
2021-12-17 17:36:39,672 [529] [WARNING ] [deprecation.py:339] From benchmark_blade_resnet50_v1.py:182: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.graph_util.extract_sub_graph`
2021-12-17 17:36:40,544 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so
2021-12-17 17:36:42,236 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_kfs45qv3_20211217173641/inputa_7rex03.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_kfs45qv3_20211217173641/outputicnikzxa.npy in subproc_run
2021-12-17 17:36:52,303 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Details:
elapsed: 3714.009, env config: {}

2021-12-17 17:36:53,374 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/input4b01gviy.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/outputqof932l2.npy in subproc_run
2021-12-17 17:36:53,374 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/input4b01gviy.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/outputqof932l2.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:36:54,457 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:36:54,457 [642] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/input4b01gviy.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/outputqof932l2.npy']

2021-12-17 17:36:54,458 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:36:54,457 [642] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/input4b01gviy.npy

2021-12-17 17:36:56,762 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:36:56,762 [642] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:36:56,769 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:36:56,769 [642] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:37:04,064 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:37:04,064 [642] [INFO    ] [subproc_runner_helper.py:9] Save profile helper results to: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/outputqof932l2.npy

2021-12-17 17:37:05,130 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 0, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/input4b01gviy.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_adtgw1pj_20211217173652/outputqof932l2.npy
2021-12-17 17:37:05,131 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Details:
elapsed: 7.698, env config: {}

2021-12-17 17:37:05,405 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Test data information, source: TestDataSource.BLADE_DEDUCED, shape variation: ShapeVariation.STATIC, info: [<blade.data.data.DataInfo object at 0x7f7feabba940>]
2021-12-17 17:37:06,086 [529] [WARNING ] [benchmark_blade_resnet50_v1.py:182] requires pygraphviz http://pygraphviz.github.io/
Failed to generate png for visualizing the graph, but it will not influence the optimization process. This issue may be caused by the missing of pygraphviz/graphviz-dev. If you want to see the png, try to install them and call the blade again.
2021-12-17 17:37:06,087 [529] [WARNING ] [benchmark_blade_resnet50_v1.py:182] requires pygraphviz http://pygraphviz.github.io/
Failed to export digraph, but it will not influence the optimization process. This issue may be caused by the missing of pygraphviz/graphviz-dev. If you want to see the digraph, try to install them and call the blade again.
2021-12-17 17:37:06,163 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfRemoveIdentityN:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:37:09,770 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Performing "/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/optimization/gpu_passes/tf_trt_plus_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_input2us75zla.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_outputf955qg4u.npy"
2021-12-17 17:37:09,770 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/optimization/gpu_passes/tf_trt_plus_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_input2us75zla.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_outputf955qg4u.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 60.0, shell mode: True
2021-12-17 17:37:12,401 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:37:12,400 [715] [INFO    ] [tf_trt_plus_helper.py:18] Load tf2trt_plus helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/optimization/gpu_passes/tf_trt_plus_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_input2us75zla.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_outputf955qg4u.npy']

2021-12-17 17:37:12,402 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:37:12,401 [715] [INFO    ] [tf_trt_plus_helper.py:18] Load tf2trt_plus helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_input2us75zla.npy

2021-12-17 17:37:13,240 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:37:13,240 [715] [WARNING ] [tf_trt_plus_helper.py:18] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:37:14,531 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:37:14,531 [715] [INFO    ] [tf_trt_plus_helper.py:18] Save tf2trt_plus subgraph results to: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_outputf955qg4u.npy

2021-12-17 17:37:15,056 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 0, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/optimization/gpu_passes/tf_trt_plus_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_input2us75zla.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/Tf2TrtPlus_6h4qxxoy_20211217173708/tf2trt_plus_outputf955qg4u.npy
2021-12-17 17:37:17,067 [529] [WARNING ] [deprecation.py:339] From /usr/local/lib/python3.6/dist-packages/tf2onnx/tf_loader.py:305: convert_variables_to_constants (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.graph_util.convert_variables_to_constants`
2021-12-17 17:37:18,939 [529] [INFO    ] [tfonnx.py:414] Using tensorflow=2.4.0, onnx=1.10.2, tf2onnx=1.9.3/1190aa
2021-12-17 17:37:18,940 [529] [INFO    ] [tfonnx.py:415] Using opset <onnx, 9>
2021-12-17 17:37:19,844 [529] [INFO    ] [tf_utils.py:278] Computed 0 values for constant folding
2021-12-17 17:37:21,186 [529] [INFO    ] [__init__.py:48] Optimizing ONNX model
2021-12-17 17:37:23,631 [529] [INFO    ] [__init__.py:83] After optimization: BatchNormalization -53 (53->0), Const -160 (268->108), GlobalAveragePool +1 (0->1), Identity -4 (4->0), ReduceMean -1 (1->0), Transpose -221 (222->1)
2021-12-17 17:37:23,654 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Fp16 optimization enabled for tensorrt in Tf2TrtPlus pass
2021-12-17 17:37:55,753 [529] [WARNING ] [benchmark_blade_resnet50_v1.py:182] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops
2021-12-17 17:37:58,162 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputgwv0p1ei.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputxj3dnbgg.npy in subproc_run
2021-12-17 17:37:58,162 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputgwv0p1ei.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputxj3dnbgg.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:37:59,307 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:37:59,306 [822] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputgwv0p1ei.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputxj3dnbgg.npy']

2021-12-17 17:37:59,307 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:37:59,307 [822] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputgwv0p1ei.npy

2021-12-17 17:38:01,623 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:01,623 [822] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:38:01,629 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:01,629 [822] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:38:08,951 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:08,951 [822] [INFO    ] [subproc_runner_helper.py:9] Save profile helper results to: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputxj3dnbgg.npy

2021-12-17 17:38:10,013 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 0, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputgwv0p1ei.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputxj3dnbgg.npy
2021-12-17 17:38:10,015 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Details:
elapsed: 7.600, env config: {}

2021-12-17 17:38:11,728 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputzpk_95bz.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputanu7ib7i.npy in subproc_run
2021-12-17 17:38:11,728 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputzpk_95bz.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputanu7ib7i.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:38:12,866 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:38:12,865 [926] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputzpk_95bz.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputanu7ib7i.npy']

2021-12-17 17:38:12,867 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:12,866 [926] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputzpk_95bz.npy

2021-12-17 17:38:15,561 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:15,561 [926] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:38:15,571 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:15,570 [926] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:38:18,251 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] [INFO] [926#926] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [926#926] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1

2021-12-17 17:38:20,699 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:20,699 [926] [INFO    ] [subproc_runner_helper.py:9] Save profile helper results to: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputanu7ib7i.npy

2021-12-17 17:38:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 0, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/inputzpk_95bz.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_ta98jgvt_20211217173757/outputanu7ib7i.npy
2021-12-17 17:38:21,380 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Details:
elapsed: 1.513, env config: {}

2021-12-17 17:38:21,387 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Tf2TrtPlus:{'kind': 'success', 'speedup': 5.024313646746791, 'pre_run_ms': 7.5999809599667785, 'post_run_ms': 1.51264063000679, 'pattern_count': 0}
2021-12-17 17:38:21,444 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TRTEngineOp op has been found, skipped tensorflow's origin tensorrt optimization.
2021-12-17 17:38:21,444 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Tf2TrtFp32:{'kind': 'skipped', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:21,468 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TRTEngineOp op has been found, skipped tensorflow's origin tensorrt optimization.
2021-12-17 17:38:21,468 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Tf2TrtFp16:{'kind': 'skipped', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:21,468 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] CombinedSwitch:{'kind': 'performace_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:21,469 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] CombinedSequence:{'kind': 'success', 'speedup': 5.024313646746791, 'pre_run_ms': 7.5999809599667785, 'post_run_ms': 1.51264063000679, 'pattern_count': 0}
2021-12-17 17:38:21,809 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so
2021-12-17 17:38:21,835 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Graph transformations:
['remove_nodes(op=Identity)']
2021-12-17 17:38:24,144 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Found 0 patterns
2021-12-17 17:38:24,173 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfFuseEmbedding:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:24,219 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] transform: strip_unused_nodes
2021-12-17 17:38:26,388 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfStripUnusedNodes:{'kind': 'success', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:29,064 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfFoldConstants:{'kind': 'success', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:29,144 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfStaticBilstmOpt:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:29,188 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Try to optimize Dynamic LSTM ops.
2021-12-17 17:38:29,429 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Graph transformations:
['remove_nodes(op=Identity)']
2021-12-17 17:38:31,984 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfLstmToCudnnRnn:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:32,006 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Try to optimize Dynamic LSTM ops.
2021-12-17 17:38:32,028 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Graph transformations:
['remove_nodes(op=Identity)']
2021-12-17 17:38:34,358 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfLstmToCudnnRnn:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:34,358 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] CombinedSwitch:{'kind': 'performace_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:34,360 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] CombinedSequence:{'kind': 'performace_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:34,406 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Found 0 optimizable Bilstm op
2021-12-17 17:38:34,407 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfCudnnrnnBilstm:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:34,472 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Graph transformations:
['remove_nodes(op=Identity)']
2021-12-17 17:38:36,607 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfFoldBatchNorms:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:36,653 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfNonMaxSuppressionOpt: total 0 NMS ops replaced with BladeFastNMS ones
2021-12-17 17:38:36,654 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfNonMaxSuppressionOpt:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:40,668 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputm_jszoxe.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputk_oop6eb.npy in subproc_run
2021-12-17 17:38:40,668 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputm_jszoxe.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputk_oop6eb.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:38:41,825 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:38:41,824 [995] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputm_jszoxe.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputk_oop6eb.npy']

2021-12-17 17:38:41,825 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:41,825 [995] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputm_jszoxe.npy

2021-12-17 17:38:44,539 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:44,539 [995] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:38:44,548 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:44,548 [995] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:38:47,238 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] [INFO] [995#995] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [995#995] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1

2021-12-17 17:38:49,718 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:49,718 [995] [INFO    ] [subproc_runner_helper.py:9] Save profile helper results to: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputk_oop6eb.npy

2021-12-17 17:38:50,392 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 0, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputm_jszoxe.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputk_oop6eb.npy
2021-12-17 17:38:50,393 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Details:
elapsed: 1.505, env config: {}

2021-12-17 17:38:50,947 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy in subproc_run
2021-12-17 17:38:50,947 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:38:52,097 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:38:52,096 [1041] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy']

2021-12-17 17:38:52,097 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:52,097 [1041] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy

2021-12-17 17:38:53,946 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:53,946 [1041] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:38:53,950 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:53,949 [1041] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:38:55,472 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] [INFO] [1041#1041] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1041#1041] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1

2021-12-17 17:38:57,083 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:38:57.083081: F /blade_src/blade/custom_ops/trt_engine_op/trt_engine_op.cc:84] Check failed: status.ok() Instantiating native function subgraph_0 failed. Because of: Function subgraph_0 is not defined.

2021-12-17 17:38:57,181 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Aborted

2021-12-17 17:38:57,181 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 134, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy
2021-12-17 17:38:57,182 [529] [ERROR   ] [benchmark_blade_resnet50_v1.py:182] Run /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy failed with error code 134, logs:
===STDOUT===

===STDERR===
/usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:38:52,096 [1041] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy']
2021-12-17 17:38:52,097 [1041] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy
2021-12-17 17:38:53,946 [1041] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops
2021-12-17 17:38:53,949 [1041] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so
[INFO] [1041#1041] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1041#1041] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1
2021-12-17 17:38:57.083081: F /blade_src/blade/custom_ops/trt_engine_op/trt_engine_op.cc:84] Check failed: status.ok() Instantiating native function subgraph_0 failed. Because of: Function subgraph_0 is not defined.
Aborted

===LOG END===
2021-12-17 17:38:57,191 [529] [ERROR   ] [benchmark_blade_resnet50_v1.py:182] optimization pass TfAutoMixedPrecisionGpu_13 failed: ERROR 2-1-0-1: Run /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy failed with error code 134, logs:
===STDOUT===

===STDERR===
/usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:38:52,096 [1041] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy']
2021-12-17 17:38:52,097 [1041] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy
2021-12-17 17:38:53,946 [1041] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops
2021-12-17 17:38:53,949 [1041] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so
[INFO] [1041#1041] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1041#1041] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1
2021-12-17 17:38:57.083081: F /blade_src/blade/custom_ops/trt_engine_op/trt_engine_op.cc:84] Check failed: status.ok() Instantiating native function subgraph_0 failed. Because of: Function subgraph_0 is not defined.
Aborted

===LOG END===
Traceback (most recent call last):
  File "blade/optimization/optimization_pass.py", line 248, in blade.optimization.optimization_pass.CombinedSequence.optimize
  File "blade/optimization/optimization_pass.py", line 70, in blade.optimization.optimization_pass.OptimizationPass._run_optimization_result
  File "blade/optimization/gpu_passes/tf_auto_mixed_precision_gpu.py", line 41, in blade.optimization.gpu_passes.tf_auto_mixed_precision_gpu.TfAutoMixedPrecisionGpu.optimize
  File "blade/optimization/gpu_passes/tf_auto_mixed_precision_gpu.py", line 68, in blade.optimization.gpu_passes.tf_auto_mixed_precision_gpu.TfAutoMixedPrecisionGpu.optimize
  File "blade/profiling/common_profiler.py", line 60, in blade.profiling.common_profiler.profile
  File "blade/profiling/profiler.py", line 156, in blade.profiling.profiler.Profiler.run
  File "blade/profiling/subproc_profiler.py", line 60, in blade.profiling.subproc_profiler.SubProcProfiler._profile
  File "blade/profiling/subproc_profiler.py", line 49, in blade.profiling.subproc_profiler.subproc_run
blade.errors.BladeInternalError: ERROR 2-1-0-1: Run /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy failed with error code 134, logs:
===STDOUT===

===STDERR===
/usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:38:52,096 [1041] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/outputo2okuozb.npy']
2021-12-17 17:38:52,097 [1041] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_a31u1d6a_20211217173838/inputkj117r3v.npy
2021-12-17 17:38:53,946 [1041] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops
2021-12-17 17:38:53,949 [1041] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so
[INFO] [1041#1041] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1041#1041] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1
2021-12-17 17:38:57.083081: F /blade_src/blade/custom_ops/trt_engine_op/trt_engine_op.cc:84] Check failed: status.ok() Instantiating native function subgraph_0 failed. Because of: Function subgraph_0 is not defined.
Aborted

===LOG END===
2021-12-17 17:38:57,192 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfAutoMixedPrecisionGpu:{'kind': 'blade_optimization_pass_exception', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:38:57,499 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Run TfAicompilerGpuPass with config: {'enable': True, 'enable_bace': False, 'enable_control_flow': False, 'optimize_for_static_shape': False, 'optimize_for_dynamic_shape': True, 'enable_eager_compilation': True, 'enable_static_shape_exhausted_fusion': False}
2021-12-17 17:38:58,898 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/input6j0aeumn.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputf6061ags.npy in subproc_run
2021-12-17 17:38:58,899 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/input6j0aeumn.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputf6061ags.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:39:00,052 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:39:00,051 [1088] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/input6j0aeumn.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputf6061ags.npy']

2021-12-17 17:39:00,052 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:00,052 [1088] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/input6j0aeumn.npy

2021-12-17 17:39:02,780 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:02,780 [1088] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:39:02,789 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:02,789 [1088] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:39:05,489 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] [INFO] [1088#1088] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1088#1088] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1

2021-12-17 17:39:07,939 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:07,939 [1088] [INFO    ] [subproc_runner_helper.py:9] Save profile helper results to: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputf6061ags.npy

2021-12-17 17:39:08,614 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 0, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/input6j0aeumn.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputf6061ags.npy
2021-12-17 17:39:08,616 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Details:
elapsed: 1.503, env config: {}

2021-12-17 17:39:10,120 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/inputcrvj9_5g.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputi491fpzr.npy in subproc_run
2021-12-17 17:39:10,120 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/inputcrvj9_5g.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputi491fpzr.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:39:11,269 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:39:11,269 [1133] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/inputcrvj9_5g.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputi491fpzr.npy']

2021-12-17 17:39:11,270 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:11,269 [1133] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/inputcrvj9_5g.npy

2021-12-17 17:39:13,964 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:13,964 [1133] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:39:13,973 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:13,973 [1133] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:39:14,046 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:14,046 [1133] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/util/../lib/libtao_ops.so

2021-12-17 17:39:16,687 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] [INFO] [1133#1133] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1133#1133] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1

2021-12-17 17:39:19,114 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:19,114 [1133] [INFO    ] [subproc_runner_helper.py:9] Save profile helper results to: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputi491fpzr.npy

2021-12-17 17:39:19,787 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 0, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/inputcrvj9_5g.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_fgi9c6ag_20211217173857/outputi491fpzr.npy
2021-12-17 17:39:19,789 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Details:
elapsed: 1.502, env config: {'BRIDGE_ENABLE_TAO': 'true', 'TAO_COMPILER_PATH': '/usr/local/lib/python3.6/dist-packages/blade/util/../lib/tao_compiler_main', 'TAO_COMPILATION_MODE_ASYNC': 'false', 'TAO_ENABLE_MLIR': 'true', 'TAO_MLIR_BRANCH_ONLY': 'true', 'TAO_EXPERIMENTAL_ENABLE_MLIR_WHOLE_GRAPH_COMPILATION': 'true'}

2021-12-17 17:39:19,793 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfAicompilerGpu:{'kind': 'performace_failed', 'speedup': 1.0007276984179434, 'pre_run_ms': 1.5027036799490452, 'post_run_ms': 1.5016109600290657, 'pattern_count': 0}
2021-12-17 17:39:19,795 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] CombinedSequence:{'kind': 'performace_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:19,865 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Optimize 0 patterns of LSTM element wise ops
2021-12-17 17:39:19,865 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfLstmElementWiseOps:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:19,911 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Optimize 0 [shape->Gather->prod->stack](double=link) patterns by placing on host.
2021-12-17 17:39:19,911 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Optimize 0 [shape->GatherV2->prod->stack](double=link) patterns by placing on host.
2021-12-17 17:39:19,911 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Optimize 0 [shape->Gather->prod->stack](single-link) patterns by placing on host.
2021-12-17 17:39:19,911 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Optimize 0 [shape->GatherV2->prod->stack](single-link) patterns by placing on host.
2021-12-17 17:39:19,911 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Optimized by moving 0 ops to Host.
2021-12-17 17:39:19,911 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfNodePlacement:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:19,935 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Try to optimize Fused Transformer Layernorm ops.
2021-12-17 17:39:21,282 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfFuseNmtLayerNorms:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:21,307 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Try to Fuse Encoder Attention Pattern.
2021-12-17 17:39:21,330 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [BladeAttentionReshapeTranspose] was found.
2021-12-17 17:39:21,330 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [BladeAttentionReshapeTranspose] pattern.
2021-12-17 17:39:21,330 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [Softmax] was found.
2021-12-17 17:39:21,330 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [Softmax] pattern.
2021-12-17 17:39:21,330 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [BladeFusedTransformerEncoderAttention] was found.
2021-12-17 17:39:21,330 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [BladeFusedTransformerEncoderAttention] pattern.
2021-12-17 17:39:21,330 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Level 2 Optimized 0 Patterns.
2021-12-17 17:39:21,330 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfFuseNmtEncoderAttentions:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:21,354 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Try to optimize Transformer Decoder Attention Pattern.
2021-12-17 17:39:21,376 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [BladeAttentionReshapeTranspose] was found.
2021-12-17 17:39:21,376 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [BladeAttentionReshapeTranspose] pattern.
2021-12-17 17:39:21,376 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [Softmax] was found.
2021-12-17 17:39:21,376 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [Softmax] pattern.
2021-12-17 17:39:21,376 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [GatherNdGatherNd] was found.
2021-12-17 17:39:21,376 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [GatherNdGatherNd] pattern.
2021-12-17 17:39:21,376 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [UnpackPackReshape] was found.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [UnpackPackReshape] pattern.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [TransformerDecoderWing] was found.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [TransformerDecoderWing] pattern.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [BladeFusedTransformerDecoderAttention] was found.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [BladeFusedTransformerDecoderAttention] pattern.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Level 2 Optimized 0 Patterns.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 0
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 0
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [BladeAttentionReshapeTranspose] was found.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [BladeAttentionReshapeTranspose] pattern.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [Softmax] was found.
2021-12-17 17:39:21,377 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [Softmax] pattern.
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [GatherNdGatherNdV1] was found.
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [GatherNdGatherNdV1] pattern.
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [TransformerDecoderWingV1] was found.
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [TransformerDecoderWingV1] pattern.
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] No candidate pattern for [BladeFusedTransformerDecoderAttentionV1] was found.
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] None Optimization for [BladeFusedTransformerDecoderAttentionV1] pattern.
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Level 2 Optimized 0 Patterns.
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 0
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 0
2021-12-17 17:39:21,378 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfFuseNmtDecoderAttentions:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:21,403 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Try to optimize batch matmul ops.
2021-12-17 17:39:22,201 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfOptimizeBatchMatmul:{'kind': 'pattern_matching_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:25,745 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/inputy2a01ivk.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputj4pkri2i.npy in subproc_run
2021-12-17 17:39:25,746 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/inputy2a01ivk.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputj4pkri2i.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:39:26,887 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:39:26,886 [1191] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/inputy2a01ivk.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputj4pkri2i.npy']

2021-12-17 17:39:26,887 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:26,887 [1191] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/inputy2a01ivk.npy

2021-12-17 17:39:29,586 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:29,586 [1191] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:39:29,595 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:29,595 [1191] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:39:32,259 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] [INFO] [1191#1191] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1191#1191] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1

2021-12-17 17:39:34,735 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:34,735 [1191] [INFO    ] [subproc_runner_helper.py:9] Save profile helper results to: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputj4pkri2i.npy

2021-12-17 17:39:35,403 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 0, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/inputy2a01ivk.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputj4pkri2i.npy
2021-12-17 17:39:35,405 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Details:
elapsed: 1.500, env config: {}

2021-12-17 17:39:35,793 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy in subproc_run
2021-12-17 17:39:35,794 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:39:36,938 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:39:36,937 [1235] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy']

2021-12-17 17:39:36,938 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:36,938 [1235] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy

2021-12-17 17:39:38,801 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:38,800 [1235] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:39:38,804 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:38,804 [1235] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:39:40,316 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] [INFO] [1235#1235] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1235#1235] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1

2021-12-17 17:39:41,912 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:41.912291: F /blade_src/blade/custom_ops/trt_engine_op/trt_engine_op.cc:84] Check failed: status.ok() Instantiating native function subgraph_0 failed. Because of: Function subgraph_0 is not defined.

2021-12-17 17:39:42,010 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Aborted

2021-12-17 17:39:42,011 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 134, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy
2021-12-17 17:39:42,011 [529] [ERROR   ] [benchmark_blade_resnet50_v1.py:182] Run /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy failed with error code 134, logs:
===STDOUT===

===STDERR===
/usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:39:36,937 [1235] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy']
2021-12-17 17:39:36,938 [1235] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy
2021-12-17 17:39:38,800 [1235] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops
2021-12-17 17:39:38,804 [1235] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so
[INFO] [1235#1235] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1235#1235] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1
2021-12-17 17:39:41.912291: F /blade_src/blade/custom_ops/trt_engine_op/trt_engine_op.cc:84] Check failed: status.ok() Instantiating native function subgraph_0 failed. Because of: Function subgraph_0 is not defined.
Aborted

===LOG END===
2021-12-17 17:39:42,019 [529] [ERROR   ] [benchmark_blade_resnet50_v1.py:182] optimization pass TfAutoMixedPrecisionGpu_22 failed: ERROR 2-1-0-1: Run /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy failed with error code 134, logs:
===STDOUT===

===STDERR===
/usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:39:36,937 [1235] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy']
2021-12-17 17:39:36,938 [1235] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy
2021-12-17 17:39:38,800 [1235] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops
2021-12-17 17:39:38,804 [1235] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so
[INFO] [1235#1235] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1235#1235] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1
2021-12-17 17:39:41.912291: F /blade_src/blade/custom_ops/trt_engine_op/trt_engine_op.cc:84] Check failed: status.ok() Instantiating native function subgraph_0 failed. Because of: Function subgraph_0 is not defined.
Aborted

===LOG END===
Traceback (most recent call last):
  File "blade/optimization/optimization_pass.py", line 248, in blade.optimization.optimization_pass.CombinedSequence.optimize
  File "blade/optimization/optimization_pass.py", line 70, in blade.optimization.optimization_pass.OptimizationPass._run_optimization_result
  File "blade/optimization/gpu_passes/tf_auto_mixed_precision_gpu.py", line 41, in blade.optimization.gpu_passes.tf_auto_mixed_precision_gpu.TfAutoMixedPrecisionGpu.optimize
  File "blade/optimization/gpu_passes/tf_auto_mixed_precision_gpu.py", line 68, in blade.optimization.gpu_passes.tf_auto_mixed_precision_gpu.TfAutoMixedPrecisionGpu.optimize
  File "blade/profiling/common_profiler.py", line 60, in blade.profiling.common_profiler.profile
  File "blade/profiling/profiler.py", line 156, in blade.profiling.profiler.Profiler.run
  File "blade/profiling/subproc_profiler.py", line 60, in blade.profiling.subproc_profiler.SubProcProfiler._profile
  File "blade/profiling/subproc_profiler.py", line 49, in blade.profiling.subproc_profiler.subproc_run
blade.errors.BladeInternalError: ERROR 2-1-0-1: Run /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy failed with error code 134, logs:
===STDOUT===

===STDERR===
/usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:39:36,937 [1235] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/outputgbezh0w5.npy']
2021-12-17 17:39:36,938 [1235] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_profile_qo10vf8r_20211217173924/input99vrnz16.npy
2021-12-17 17:39:38,800 [1235] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops
2021-12-17 17:39:38,804 [1235] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so
[INFO] [1235#1235] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1235#1235] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1
2021-12-17 17:39:41.912291: F /blade_src/blade/custom_ops/trt_engine_op/trt_engine_op.cc:84] Check failed: status.ok() Instantiating native function subgraph_0 failed. Because of: Function subgraph_0 is not defined.
Aborted

===LOG END===
2021-12-17 17:39:42,020 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] TfAutoMixedPrecisionGpu:{'kind': 'blade_optimization_pass_exception', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:42,020 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] CombinedSequence:{'kind': 'performace_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:42,020 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] CombinedSwitch:{'kind': 'performace_failed', 'speedup': None, 'pre_run_ms': None, 'post_run_ms': None, 'pattern_count': 0}
2021-12-17 17:39:44,159 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/inputc2ntz2xs.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/output_7uf3ula.npy in subproc_run
2021-12-17 17:39:44,159 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Execute shell command: `/usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/inputc2ntz2xs.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/output_7uf3ula.npy`, cwd: /root/inference-optimization-benchmark/blade, timeout: 600.0, shell mode: True
2021-12-17 17:39:45,310 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] /usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes
2021-12-17 17:39:45,309 [1280] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: ['/usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/inputc2ntz2xs.npy', '/root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/output_7uf3ula.npy']

2021-12-17 17:39:45,310 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:45,310 [1280] [INFO    ] [subproc_runner_helper.py:9] Load profile helper args from: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/inputc2ntz2xs.npy

2021-12-17 17:39:48,024 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:48,024 [1280] [WARNING ] [subproc_runner_helper.py:9] failed to import tf_contrib_module_op_def_2_4_0.py on error 'No module named 'blade.model.tf_contrib_module_op_def_2_4_0''. It may meet errors if the input graph contains tf contrib ops

2021-12-17 17:39:48,033 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:48,033 [1280] [INFO    ] [subproc_runner_helper.py:9] Loading TF Custom Op:/usr/local/lib/python3.6/dist-packages/blade/lib/libblade_ops.so

2021-12-17 17:39:50,721 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] [INFO] [1280#1280] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:50] Linked TensorRT version: 8.0.1
[INFO] [1280#1280] [/blade_src/blade/tensorrt/bridge/tensorrt_common.cpp:51] Loaded TensorRT version: 8.0.1

2021-12-17 17:39:53,222 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] 2021-12-17 17:39:53,221 [1280] [INFO    ] [subproc_runner_helper.py:9] Save profile helper results to: /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/output_7uf3ula.npy

2021-12-17 17:39:53,892 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] retcode: 0, command: /usr/bin/python3 /usr/local/lib/python3.6/dist-packages/blade/profiling/subproc_runner_helper.py /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/inputc2ntz2xs.npy /root/inference-optimization-benchmark/blade/20211217173629_f0myd79j/subproc_run_mtd3gmbp_20211217173942/output_7uf3ula.npy
2021-12-17 17:39:53,893 [529] [INFO    ] [benchmark_blade_resnet50_v1.py:182] Details:
elapsed: 1.503, env config: {}

2021-12-17 17:39:54,241 [529] [WARNING ] [deprecation.py:339] From benchmark_blade_resnet50_v1.py:196: FastGFile.__init__ (from tensorflow.python.platform.gfile) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.gfile.GFile.
